tests/tracker_test.go

```diff
@@ -1,9 +1,29 @@
 package easypost_test
 
 import (
+	"strconv"
+
 	"github.com/EasyPost/easypost-go"
 )
 
+func (c *ClientTests) TestTrackerCreateList() {
+	client := c.TestClient()
+	assert := c.Assert()
+
+	trackingCodeParam := make(map[string]interface{})
+	trackingCodes := []string{"EZ1000000001", "EZ1000000001", "EZ1000000001"}
+
+	for i := range trackingCodes {
+		trackingCodeParam[strconv.Itoa(i)] = map[string]string{
+			"tracking_code": trackingCodes[i],
+			"carrier":       "USPS",
+		}
+	}
```

Comment:
Although this seems to work, it actually needs to allow for a map of maps allowing both a `tracking_code` parameter and a `carrier` parameter. This looks like it's taking an array of `tracking_code` which isn't correct. It needs to be individual maps of both `tracking_code` and `carrier`.

Reply:
The reason this is important is this is a bulk tracking endpoint. It won't always be just one carrier. Someone may want to create 100 trackers from 10 different carriers and each tracking code would need to allow its own carrier to be associated for that reason. This looks like it only allows for a single carrier for the whole batch.

Reply:
Nevermind, I misread this.